src/renderer/widgets/IWidget.cpp

```diff
@@ -100,21 +100,26 @@ static void replaceAllLayout(std::string& str) {
     }
 }
 
+static const auto  PCURRENTTZ = std::chrono::current_zone();
 static std::string getTime() {
-    const auto current_zone = std::chrono::current_zone();
-    const auto HHMMSS       = std::chrono::hh_mm_ss{current_zone->to_local(std::chrono::system_clock::now()) -
-                                              std::chrono::floor<std::chrono::days>(current_zone->to_local(std::chrono::system_clock::now()))};
-    const auto HRS          = HHMMSS.hours().count();
-    const auto MINS         = HHMMSS.minutes().count();
+    if (!PCURRENTTZ)
+        return "TIMEZONE UNKNOWN";
```

Comment:
JUST RETURN UTC

Reply:
MAYBE LOG IT ONCE

Reply:
> JUST RETURN UTC

Reasonable, done.

> MAYBE LOG IT ONCE

Turns out the current_zone pointer is not guarantied to be static i from looking at the sources. So it is much easier to always log it. I think it's fine to spam this once in ~30 secs. current_zone being a nullptr shouldn't happen really.